angular.json

```diff
@@ -26,7 +26,6 @@
               "src/favicon.ico"
             ],
             "styles": [
-              "node_modules/bootstrap/dist/css/bootstrap.css",
```

Comment:
probably we still need it in the playground for test cases when we test nebular-bootstrap?

Reply:
should we add normalize instead?